lib/_debugger.js

```diff
@@ -1696,42 +1661,42 @@ Interface.prototype.trySpawn = function(cb) {
 
   this.pause();
 
-  const client = self.client = new Client();
+  const client = this.client = new Client();
   var connectionAttempts = 0;
 
-  client.once('ready', function() {
-    self.stdout.write(' ok\n');
+  client.once('ready', () => {
+    this.stdout.write(' ok\n');
 
     // Restore breakpoints
-    breakpoints.forEach(function(bp) {
-      self.print('Restoring breakpoint ' + bp.scriptReq + ':' + bp.line);
-      self.setBreakpoint(bp.scriptReq, bp.line, bp.condition, true);
+    breakpoints.forEach((bp) => {
+      this.print('Restoring breakpoint ' + bp.scriptReq + ':' + bp.line);
+      this.setBreakpoint(bp.scriptReq, bp.line, bp.condition, true);
```

Comment:
Not 100% sure here. Did you test if nested arrow func correctly bind `this` to the outermost scope?